src/js/components/Spinner/Spinner.js

```diff
@@ -53,6 +54,21 @@ const Spinner = forwardRef(
     const color = colorProp || colorThemeProp;
     const Icon = theme.spinner.icon;
 
+    const defaultBorder = [
+      { side: 'all', color: 'background-contrast', size },
+      { side: 'top', color, size },
+    ];
+    const spinnerBorder = Array.isArray(borderThemeProp)
+      ? borderThemeProp.map((borderSide) => ({
+          ...borderSide,
+          color:
+            borderSide.side === 'all'
```

Comment:
Are we able to cover this with an automated test?

Reply:
@wenderjean I'm not entirely sure, but this seems to be covered on this spec: https://github.com/grommet/grommet/blob/master/src/js/components/Spinner/__tests__/Spinner-test.js#L77

Reply:
We should have an automated test with a theme and the color prop defined on spinner